src/main/java/com/snack/news/dto/AdminNewsDto.java

```diff
@@ -5,13 +5,7 @@
 import com.snack.news.domain.news.News;
 import com.snack.news.domain.tag.Tag;
 import com.snack.news.domain.topic.Topic;
-import com.snack.news.domain.topic.TopicType;
-import lombok.Getter;
-import lombok.Setter;
-import lombok.ToString;
-import lombok.NoArgsConstructor;
-import lombok.AllArgsConstructor;
-import lombok.Builder;
+import lombok.*;
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Avoid unused imports such as 'lombok'](https://app.codacy.com/manual/snack-news/Snack-BE/pullRequest?prid=4962836)